cypress/e2e/tests/migration/applicationinventory/analysis/custom-rules.test.ts

```diff
@@ -130,6 +130,23 @@ describe(["@tier2"], "Bug MTA-2015: Custom Rules in analyses", function () {
         app.verifyFileNotValidXML();
     });
 
+    it("Python custom rules file analysis", function () {
+        const app = new Analysis(
+            getRandomApplicationData("python-app-custom-rules", {
+                sourceData: this.appData["python-demo-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["python_demo_application"])
+        );
+        tackleTestappName = app.name;
+        Application.open();
```

Comment:
`application.create()` already performs `Application.open()` so this line can be safely removed

Reply:
done